include/rnp/rnp.h

```diff
@@ -2707,6 +2707,17 @@ RNP_API rnp_result_t rnp_op_verify_get_file_info(rnp_op_verify_t op,
                                                  char **         filename,
                                                  uint32_t *      mtime);
 
+/**
+ * @brief Get format of the data stored in the message, if available.
+ *
+ * @param op opaque verification context. Must be initialized and have execute() called on it.
+ * @param format character describing format would be stored here, see RFC 4880 section 5.9 and
+ *               further standard extensions for possible values. If information is not
+ *               available then \0 value will be stored here. Cannot be NULL.
```

Comment:
Sorry fir that note but:
`\0' - if I am getting char value
\0 - if  int value

Reply:
Thanks for spotting, fixed and re-pushed.